build/ci/evergreen.yml

```diff
@@ -65,6 +68,33 @@ functions:
       params:
         <<: *go_options
         command: make check-library-owners
+  "fetch deps cache":
+    - command: subprocess.exec
+      type: setup
+      params:
+        <<: *go_options
+        binary: build/ci/fetch-cache-deps.sh
+    - command: expansions.update
+      params:
+        file: src/github.com/mongodb/mongodb-atlas-cli/sha_expansion.yaml
+  "save deps cache":
+    - command: subprocess.exec
+      type: setup
+      params:
+        <<: *go_options
+        binary: build/ci/save-cache-deps.sh
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: src/github.com/mongodb/mongodb-atlas-cli/gomod-${go_dep_sha}.tar.xz
+        remote_file: dependencies/go/gomod-${go_dep_sha}.tar.xz
+        bucket: mongodb-mongocli-build
```

Comment:
do we have to worry about the storage (each [cache](https://spruce.mongodb.com/task/mongocli_master_code_health_compile_atlascli_patch_b2654d4d095239737b777eb36cc4adcb2f8a282c_64076effc9ec441d7c25c546_23_03_07_17_06_08/files?execution=0) is > 200mb) or this bucket has a policy of automatically discarding old files?

Reply:
replying off line